packages/zipkin-transport-http/package.json

```diff
@@ -11,14 +11,12 @@
   "author": "Openzipkin <openzipkin.alt@gmail.com>",
   "license": "Apache-2.0",
   "repository": "https://github.com/openzipkin/zipkin-js",
-  "dependencies": {
-    "node-fetch": "^1.5.3"
-  },
   "devDependencies": {
-    "babel-cli": "^6.23.0",
+    "babel-cli": "^6.26.0",
     "body-parser": "^1.15.2",
     "express": "^4.13.4",
     "mocha": "^3.2.0",
+    "node-fetch": "^1.5.3",
```

Comment:
this should still be a dep?

Reply:
maybe https://github.com/matthew-andrews/isomorphic-fetch is better, if it works with react-native?

Reply:
fetch-ponyfill doesn't modify the global environment, so that's probably a better alternative